contrib/python/api/skydive/packet_injector.py

```diff
@@ -22,8 +22,8 @@ class PacketInjection(object):
 
     def __init__(self, uuid="", src="", dst="", srcip="", dstip="", srcmac="",
                  dstmac="", srcport="", dstport="", type="", payload="",
-                 trackingid="", icmpid="", count="", interval="",
-                 increment="", starttime=""):
+                 trackingid="", icmpid="", count=1, interval=0,
+                 increment=0, starttime="", ttl=255):
```

Comment:
See the comment of @hunchback, better to use 64

Reply:
Sorry @masco-enovance my bad `Increment` should be set to `False` per the API definition